gui/GameConqueror.xml

```diff
@@ -124,18 +134,7 @@ along with GameConqueror.  If not, see <http://www.gnu.org/licenses/>.
               </packing>
             </child>
             <child>
-              <object class="GtkAlignment" id="alignment1">
```

Comment:
This alignment was here for consistency (and "future-proofing"), if you want it deleted there are a couple others at end grid that should go, too.